google_photos_takeout_helper/__main__.py

```diff
@@ -236,7 +236,7 @@ def set_creation_date_from_exif(file):
                 break
             except KeyError:
                 pass
-        if datetime_str is None:
+        if datetime_str is None or datetime_str == '':
```

Comment:
Also skip if isn't blank (only whitespace):


```suggestion
        if datetime_str is None or datetime_str.strip() == '':
```